pkg/proxy/userspace/proxier.go

```diff
@@ -687,12 +696,16 @@ func iptablesCommonPortalArgs(destIP net.IP, destPort int, protocol api.Protocol
 		args = append(args, "-d", fmt.Sprintf("%s/32", destIP.String()))
 	}
 
+	if excludeContainerSourceTraffic {
+		args = append(args, "-m", "physdev", "--physdev-is-in")
```

Comment:
yes, I am missing the !
My understanding of physdev-is-in was wrong.
I thought it was for non-virtual interface (hence the term physdev)
but it seems to be true for bridged/virtual interfaces.